<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MalzemeStokFormu</class>
 <widget class="QDialog" name="MalzemeStokFormu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stok Durumu</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButtonGoster">
      <property name="text">
       <string>Stok Hesapla</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter">
     <widget class="QLineEdit" name="lineEdit_8"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButtonYenile">
      <property name="text">
       <string>Stok Temizle</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2" alignment="Qt::AlignmentFlag::AlignBottom">
     <widget class="QTableWidget" name="tableWidget_stok">
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Ürün Adı</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Toplam Giriş</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Toplam Çıkış</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Stok</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignRight">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Ürün ara :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButtonYazdir">
      <property name="text">
       <string>Yazdır</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
